akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2Blueprint.scala

```diff
@@ -79,10 +79,7 @@ object Http2Blueprint {
     // This is master header parser, every other usage should do .createShallowCopy()
     // HttpHeaderParser is not thread safe and should not be called concurrently,
     // the internal trie, however, has built-in protection and will do copy-on-write
-    val masterHttpHeaderParser = HttpHeaderParser(parserSettings, log) { info ⇒
-      if (parserSettings.illegalHeaderWarnings)
-        logParsingError(info withSummaryPrepended "Illegal request header", log, parserSettings.errorLoggingVerbosity)
-    }
+    val masterHttpHeaderParser = HttpHeaderParser(parserSettings, log)
```

Comment:
After the change the possibly logged parsing error will be `Illegal request header` and it was `Illegal response header`. If it is not possible to differentiate in the `defaultIllegalHeaderHandler` then we should just leave `illegal header` here.